modules/integrate-splunk-configure-splunk.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-splunk.adoc
+:_module-type: PROCEDURE
+[id="integrate-splunk-configure-splunk_{context}"]
+= Configuring Splunk
+
+Add a new HTTP event collector for your Splunk instance, and get the token.
+
+.Procedure
+. In your Splunk dashboard, navigate to *Settings* -> *Add Data*.
+. Click *Monitor*.
+. On the *Add Data* page, click *HTTP Event Collector*.
+. Enter a *Name* for the event collector and then click *Next >*.
+. Accept the default *Input Settings* and click *Review >*.
+. Review the event collector properties and click *Submit >*.
+. Copy the *Token Value* for the event collector.
```

Comment:
Is this the token value that will be used in another procedure? Might be helpful to mention that here, if so.